### DISCUSSION TITLE
Zigbee2mqtt and openhab

### DISCUSSION THREAD

**Discussion by dany-69** [Koenkk/zigbee2mqtt#7715] [General]
Hello, one question ...
I have configured some devices with zigbee2mqtt, on browser and mqtt Eclipse I see them but I don't know how to combine them in openhab?
I also looked in the community of oh but nothing.
Who has a guide or a file to study it?
Thank you

**Comment by sjorge** [comment]
AFAIK there is no 'auto' discovery in OpenHAB for zigbee2mqtt, you will need to manually create the things from the mqtt bindings.

e.g. as described here https://community.openhab.org/t/openhab-and-zigbee2mqtt-tutorial-for-beginners/83446

**Reply by dany-69** [reply]
how you do it? the problem is that I don't know the procedure, I'm looking for an example file where I change the parameters with my devices.
doing so I learn the logic

**Reply by sjorge** [reply]
I'm not using OpenHAB, I looked at it briefly but found the whole mqtt binding + manual config a mess and gave up. So the link above is all I found in my notes.

**Reply by dany-69** [reply]
Thanks

**Comment by dany-69** [comment]
kindly, if I send you the parameters of the magnetic contact, you could make me the c9mandi that I have to write in openhab and, above all, where I have to write them
